README.md

```diff
@@ -0,0 +1,194 @@
+
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Welcome this is Blog App Rails</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Blog-app-Rails] <a name="about-project"></a>
+
+> This is a blog app built with Ruby on Rails. It allows users to create an account, create posts, and comment on other posts.
+
+## 🛠 Built With <a name="built-with"></a>
+
++ Ruby
++ Ruby on Rails
++ PostgreSQL
++ Rspec (for testing)
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Users can create an account
+- Users can create posts
+- Users can comment on posts
+- Users can like posts
+- Users can follow other users
+- Users can see posts from users they follow
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> [Live Demo Link](https://blog-app-rails.herokuapp.com/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup <a name="setup"></a>
+
+- Clone the repository with `git clone
+
```

Comment:
- [ ] Kindly add proper command for cloning the repository :blush: as shown below :arrow_down: to make your readme descriptive :+1:

**- Clone the repository with `git clone git@github.com:andym80/Blog-app-Rails.git`**